src/open_inwoner/openklant/services.py

```diff
@@ -1209,6 +1209,41 @@ def create_question(
 
         return OpenKlant2Question.from_klantcontact_and_answer(klantcontact)
 
+    def create_question_for_zaak(
+        self,
+        partij_uuid: str,
+        question: str,
+        subject: str,
+        zaak: str,
+    ) -> OpenKlant2Question:
+        question = self.create_question(
```

Comment:
`question` shadows the arg, perhaps:

```suggestion
        ok2_question = self.create_question(
```